Fix script crashing with precedence error

agejevasv opened the pull request:
On Perl v5.42.0 the process exits with status 255 and error: 
> Possible precedence problem between ! and string eq at pt-online-schema-change line 10633.

agejevasv committed:
Fix script crashing with precedence error

On Perl v5.42.0 the process exits with status 255 and error: Possible precedence problem between ! and string eq at pt-online-schema-change line 10633.

it-percona-cla commented:
[![CLA assistant check](https://cla-assistant.percona.com:443/pull/badge/signed)](https://cla-assistant.percona.com:443/percona/percona-toolkit?pullRequest=989) <br/>All committers have signed the CLA.

svetasmirnova committed:
PR-989 -  Fix script crashing with precedence error

- Fixed typo in the tool code
- Added more diagnostic information into test for pt-1919
